src/Time/TimePickerModal.tsx

```diff
@@ -70,8 +70,8 @@ export function TimePickerModal({
   onDismiss: () => any
   onConfirm: (hoursAndMinutes: { hours: number; minutes: number }) => any
   animationType?: 'slide' | 'fade' | 'none'
-  keyboardIcon?: string
-  clockIcon?: string
+  keyboardIcon?: IconSource
```

Comment:
Can you add this change in the Docs page?
https://web-ridge.github.io/react-native-paper-dates/docs/time-picker

And also if you change this type, you

Reply:
> Can you add this change in the Docs page? https://web-ridge.github.io/react-native-paper-dates/docs/time-picker
> 
> And also if you change this `string` type, you should change this too:
> 
> ![image](https://user-images.githubusercontent.com/19764334/279498502-cc9512bc-3c94-47e6-8e0a-d3767b136b88.png)
> 
> Thanks :)

no need to change this string type,you can read code again

Reply:
![image](https://github.com/web-ridge/react-native-paper-dates/assets/66519168/03ba5b1f-db92-4f41-aa28-659d1c2f0dbf)
here icon is IconButton props,
![image](https://github.com/web-ridge/react-native-paper-dates/assets/66519168/47fcfff4-3dc0-4f4f-a14e-cf051e907943)

Reply:
> > Can you add this change in the Docs page? https://web-ridge.github.io/react-native-paper-dates/docs/time-picker
> > And also if you change this `string` type, you should change this too:
> > ![image](https://user-images.githubusercontent.com/19764334/279498502-cc9512bc-3c94-47e6-8e0a-d3767b136b88.png)
> > Thanks :)
> 
> no need to change this string type,you can read code again

Are you sure? 

![image-1](https://github.com/web-ridge/react-native-paper-dates/assets/19764334/bafb7d31-10c3-417d-940c-834ec8a519d6)

Reply:
very sorry for that,
![image](https://github.com/web-ridge/react-native-paper-dates/assets/66519168/963a778b-a01a-4f19-91cf-9a99046a47d2)